client/src/components/Hours.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Hours extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      hoursToggle: true,
+    };
+  }
+
+  handleHoursToggle(e) {
+    e.preventDefault();
+    const { hoursToggle } = this.state;
+    this.setState({
+      hoursToggle: !hoursToggle,
+    });
+  }
+
+  render() {
+    const { hoursToggle } = this.state;
+    const { openHours, closeHours } = this.props;
```

Comment:
You can destructure this portion as:

const { 
  openHours: { MonOH, TuesOH, WedOH, ThursOH, FriOH, SatOH, SunOH },
 closeHours: { MonCH, TuesCH, WedCH, ThursCH, FriCH, SatCH, SunCH },
         } = this.props

Reply:
Oh! I didn't know I can do it like that too! Thanks!